react/features/chat/components/native/MessageContainer.js

```diff
@@ -36,6 +36,7 @@ export default class MessageContainer extends AbstractMessageContainer {
                 data = { this._getMessagesGroupedBySender() }
                 inverted = { true }
                 keyExtractor = { this._keyExtractor }
+                keyboardShouldPersistTaps = 'always'
```

Comment:
What does this do?

Reply:
This lets the user tap a link without having to first dismiss the keyboard (if it's not here, the first tap only dismisses the keyboard, then you have to tap again to open the link)